ktor-network/ktor-network-tls/jvm/src/io/ktor/network/tls/OID.kt

```diff
@@ -23,20 +26,21 @@ internal data class OID(val identifier: String) {
         /**
          * Algorithm OID
          */
-        val Sha1withRSAEncryption = OID("1.2.840.113549.1.1.5")
         val ECDSAwithSHA384Encryption = OID("1.2.840.10045.4.3.3")
         val ECDSAwithSHA256Encryption = OID("1.2.840.10045.4.3.2")
 
+        val RSAwithSHA512Encryption = OID("1.2.840.113549.1.1.13")
         val RSAwithSHA384Encryption = OID("1.2.840.113549.1.1.12")
         val RSAwithSHA256Encryption = OID("1.2.840.113549.1.1.11")
+        val RSAwithSHA1Encryption = OID("1.2.840.113549.1.1.5")
```

Comment:
Isn't it deprecated? Why do we support it?

Reply:
It's mandatory by spec